src/editing.md

```diff
@@ -1,463 +1,491 @@
+es ist jedoch nicht möglich, zusätzliche Kartenvorlagen zu einem Notiztyp
+hinzuzufügen, der auf dem Typ "Lückentext" basiert.
 
-    Text1 field: {{c1::Canberra was founded}} in 1913
+## Eingabe von Sonderzeichen und Akzenten
 
-    Text2 field: {{c2::Canberra}} was founded in 1913
+Alle modernen Computer unterstützen die Eingabe fremder Symbole und Akzente
+auf unterschiedliche Art und Weise. Wir empfehlen, die Tastaturbelegung der
+Sprache zu verwenden, die du lernen willst.
 
-The default cloze note type has a second field called Extra, that is
-shown on the answer side of each card. It can be used for adding some
-usage notes or extra information.
+Sprachen, die ihre eigene Schrift haben (z. B. Japanisch, Chinesisch, Russisch usw.)
+haben stets auch spezifische Tastaturbelegungen.
 
-The cloze note type is treated specially by Anki, and cannot be created
-based on a regular note type. If you wish to customize it, please make
-sure to clone the existing Cloze type instead of another type of note.
-Things like formatting can be customized, but it is not possible to add
-extra card templates to the cloze note type.
+Europäische Sprachen, die Akzente benutzen, haben oftmals auch eigene Belegungen,
+können aber auch mit der deutschen Tastaturbelegung geschrieben werden.
+Hierfür musst du zuerst den gewünschten Akzent und dann den Buchstaben eingeben,
+der den Akzent erhalten soll - z. B. ergeben <kbd>`</kbd> und <kbd>A</kbd> das Symbol à.
 
-## Inputting Foreign Characters and Accents
+Um herauszufinden, wie du fremde Tastaturbelegungen auf deinem Betriebssystem installieren
+kannst, gib in Google Suchbegriffe wie "Japanische Tastaturbelegung Mac" oder
+"Chinesische Tastaturbelegung Windows" ein.
 
-All modern computers have built-in support for typing accents and
-foreign characters, and multiple ways to go about it. The method we
-recommend is by using a keyboard layout for the language you want to learn.
+Für Sprachen mit umgekehrter Leserichtung (RTL) wird es etwas komplizierter.
+Diese [Website](http://dotancohen.com/howto/rtl_right_to_left.html) (Englisch)
+bietet einen guten Ausgangspunkt.
 
-Languages with a separate script like Japanese, Chinese, Thai, and so on,
-have their own layouts specific to that language.
+Das Toolkit, auf dem Anki's Editor aufgebaut ist, hat Schwierigkeiten mit manchen
+Eingabemethoden wie z. B. auf dem Mac das Gedrückthalten von Tasten, um akzentuierte
+Buchstaben auszuwählen, oder die Eingabe von Schriftzeichen durch Kombination der
+<kbd>Alt</kbd>-Taste und numerischer Codes auf Windows.
 
-European languages that use accents may have their own layout, but can
-often by typed on a generic "international keyboard" layout. These work
-by typing the accent, then the character you want accented - e.g. an
-apostrophe (') then the letter a (a) gives á.
+## Unicode-Normalisierung
 
-To add the international keyboard on Windows machines, please see
```

Comment:
I think this one and the following paragraphs are missing, which is at least part of the problem.

Reply:
Ah yes, that's right. I removed those because they don't really apply for a german audience..